examples/spi-ili9486-esp32-c3/src/main.rs

```diff
@@ -0,0 +1,364 @@
+#![no_std]
+#![no_main]
+
+/* --- Needed by ESP32-c3 --- */
+use esp_backtrace as _;
+use hal::{
+    clock::ClockControl,
+    peripherals::Peripherals,
+    prelude::*,
+    spi::{Spi, SpiMode},
+    timer::TimerGroup,
+    Delay, Rtc, IO,
+};
+/* -------------------------- */
+
+use embedded_graphics::{
+    // Provides the necessary functions to draw on the display
+    draw_target::DrawTarget,
+    // Provides colors from the Rgb565 color space
+    pixelcolor::Rgb565,
+    prelude::RgbColor,
```

Comment:
```suggestion
    prelude::*,
```
or if you don't want to use the prelude:
```suggestion
    pixelcolor::RgbColor,
```